tests/network/fetchlocal.c

```diff
@@ -481,3 +481,47 @@ void test_network_fetchlocal__call_progress(void)
 	git_remote_free(remote);
 	git_repository_free(repo);
 }
+
+void test_network_fetchlocal__prune_load_remote_prune_config(void)
+{
+	git_repository *repo;
+	git_remote *origin;
+  git_config *config;
```

Comment:
Sorry to nitpick, one more whitespace issue here, then I agree that this is ready for merging.